lib/rhc/commands/scp.rb

```diff
@@ -32,6 +32,22 @@ def run(_, action, local_path, remote_path)
       raise RHC::ArgumentNotValid.new("'#{action}' is not a valid argument for this command.  Please use upload or download.") unless action == 'download' || action == 'upload'
       raise RHC::FileOrPathNotFound.new("Local file, file_path, or directory could not be found.") unless File.exist?(local_path)
 
+      if options.ssh
+        warn "User specified a ssh executable"
+        if windows?
+          warn("On Windows, file transfers to/from a gear can be completed with many different third-party tools such as FileZilla or WinSCP.\n" \
+            "Alternatively, do not specify a custom ssh executable to transfer files through the rhc tool's internal ssh implementation.")
+        else
+          if action == "upload"
+            scp_cmd = "scp -S #{options.ssh} #{local_path} '#{ssh_opts[0]}@#{ssh_opts[1]}:$HOME/#{remote_path}'"
+          else
+            scp_cmd = "scp -S #{options.ssh} '#{ssh_opts[0]}@#{ssh_opts[1]}:$HOME/#{remote_path}' #{local_path}"
```

Comment:
Mi-i-ight be worth assigning `destination = '#{ssh_opts[0]}@#{ssh_opts[1]}:$HOME/#{remote_path}'` and then using that instead of repeating all that in both `scp_cmd` assignments.